README.md

```diff
@@ -1,4 +1,4 @@
-# Hydra
+# Hydra / jekyll-theme-hydra
```

Comment:
A name change here is probably not necessary. We can mention it's a gem further down